js-intro.js

```diff
@@ -5,13 +5,31 @@
 // Consider this variable:
 
 var mantra = "Be the dev"
+var arr = mantra.charAt(0)
 
 // 1a. Write the code that returns 'B' from mantra.
-
+console.log(arr)
 // 1b. Write the code that determines if there is an 'x' in mantra.
-
+function let(){
+    for(let i = 0; i < mantra.length; i++)
+    if(mantra.includes("x")){
+        return "true"
+    }else{
+        return "false"
+    }
+}
+console.log(let(mantra))
 // 1c. Write the code that determines if there is a 'v' in mantra.
-
+function lett(){
+    for(let i = 0; i < mantra.length; i++)
+    if(mantra.includes("v")){
+        return "true"
+    }else{
+        return "false"
+    }
+}
+
+console.log(lett(mantra))
```

Comment:
You have a for loop here but it isn't actually doing anything. You can delete line 24 - and line 14 in #1b.